flask_dropzone/__init__.py

```diff
@@ -375,11 +375,18 @@ def create(action='', csrf=False, action_view='', id='myDropzone', **kwargs):
         enctype="multipart/form-data"></form>''' % (action_url, id))
 
     @staticmethod
-    def style(css):
+    def style(css='', id=''):
```

Comment:
The reason why I replaced the `css` arg from positional argument to keyword argument is that I think most of us would put `id` arg in the first place instead of `css` arg normally when we're using `id` arg. We can't achieve this with the original code.

Reply:
If we encourage the user reverses the arg order, they may pass them as positional args, so we also need to force the user to pass keyword arguments. Anyway, I think just keep the `css` as positional arg will be enough since setting `id` is not a common use case, and the user can write manual style.

Reply:
OK I see, will do the update soon.